internal/common/helpers.go

```diff
@@ -36,3 +38,51 @@ func IsStringInSortedSlice(slice []string, s string) bool {
 	}
 	return false
 }
+
+// DataSizeToUint64 converts a size specified as a string in KB/KiB/MB/etc. to
+// a number of bytes represented by uint64.
+func DataSizeToUint64(size string) *uint64 {
+	// Get the number from the string
+	plain_number := regexp.MustCompile(`[[:digit:]]+`)
+	number_as_str := plain_number.FindString(size)
+	if number_as_str == "" {
+		return nil
+	}
+
+	// Parse the number into integer, be aware that the function
+	// returns int64, so it cannot hold all possible values for uint64
+	number, err := strconv.ParseInt(number_as_str, 10, 64)
+	if err != nil {
+		return nil
+	}
+	return_size := new(uint64)
+	*return_size = uint64(number)
+
+	// List of all supported units (from kB to TB and KiB to TiB)
+	supported_units := []struct {
+		re       *regexp.Regexp
+		multiple uint64
+	}{
+		{regexp.MustCompile(`^\s*[[:digit:]]+\s*kB\s*$`), 1000},
```

Comment:
I suggest calling `strings.TrimSpace` before `MatchString` to simplify the regexes.